view/frontend/web/js/pagseguro.js

```diff
@@ -254,7 +254,7 @@ RMPagSeguro.prototype.updatePaymentHashes = function(){
         var inputCcType = jQuery('input[name="payment[pagseguropro_cc_cctype]"]');
             inputCcType.val((this.brand)?this.brand.name:'');
         var inputCcIsadmin = jQuery('input[name="payment[pagseguropro_cc_isadmin]"]');
-            inputCcIsadmin.val(this.config.is_admin);        
```

Comment:
Please remove spaces and formatting from (future) pull requests. It makes the reviewing job almost impossible in some cases.